server/libs/domain/src/smart-info/smart-info.service.ts

```diff
@@ -46,4 +46,19 @@ export class SmartInfoService {
       this.logger.error(`Unable run object detection pipeline: ${asset.id}`, error?.stack);
     }
   }
+
+  async handleEncodeClip(data: IAssetJob) {
+    const { asset } = data;
+
+    if (!MACHINE_LEARNING_ENABLED || !asset.resizePath) {
+      return;
+    }
+
+    try {
+      const clip = await this.machineLearning.encodeImage({ thumbnailPath: asset.resizePath });
```

Comment:
I think we should be able to pass in a full-size image and the library will resize it appropriately, that might yield slightly better results than using an already-resized thumb?

Reply:
Yea agreed, we should have each service do the resize itself imo so they get the exact size they need for optional recognition